tests/api/test_siblings.py

```diff
@@ -320,7 +320,7 @@
             }
         ],
         'meta': {
-            'description': 'Container advisories attached to container build 710'
+            'description': 'Container advisories attached to slf4j_2-1.7.4-4.el7_4_as'
```

Comment:
This is not correct. It should be `Container advisories that contain slf4j_2-1.7.4-4.el7_4_as` or something like that.